homeassistant/components/plex/button.py

```diff
@@ -51,4 +53,5 @@ def device_info(self) -> DeviceInfo:
         return DeviceInfo(
             identifiers={(DOMAIN, self.server_id)},
             manufacturer="Plex",
+            name=self._server_name,
```

Comment:
Shouldnt this be plaxserver friendly name to match the old logic in sensors?

Reply:
Previously it was f"Scan Clients ({server_name})", now I am reversing that and the name would be `{server_name} Scan clients`. Does that answer your question?

Reply:
Previously a sensor was named as here: https://github.com/home-assistant/core/pull/98841/files#diff-2ac2f6e7fdc724f6c62c8e637030199e553652384a417e7c5d49658921894a88L77

Thay is based on the friendly name stored in the server. Now the sensor will base its name from the device. However the device is not named after the server friendly name, it is named from a config entry stored name.

Reply:
Hmm this is an interesting one, I will dive deeper into this